### PR TITLE
Update license name for EPL-2.0

### PR DESCRIPTION
**Pull request by netomi**
Resolves: python-poetry/poetry#9803

- [x] Added **tests** for changed code.
- [ ] Updated **documentation** for changed code.

Test case was removed based on suggestion of @Secrus .

Not sure what documentation I should change in this regard, so I skip it for now, but let me know if I need to update something.

### CODE REVIEW

**Comment by dimbleby** [comment]
the file you have change is autogenerated, you do not want to be changing that.

Maybe [here](https://github.com/python-poetry/poetry-core/blob/589ecb03f28b30d9a604dc13b51fba55f3981ef5/src/poetry/core/spdx/license.py#L109)

**Comment by netomi** [comment]
oh ty, will update the PR and try to understand how the json file got generated

**Comment by dimbleby** [comment]
I dont think you need to care about the autogenerated file, just revert the change to it

**Comment by netomi** [comment]
yeah, updated my PR and force-pushed, ty guys.

File: tests/spdx/test_license.py [hunk 1]
```diff
@@ -12,6 +12,12 @@ def test_classifier_name() -> None:
     )
 
 
+def test_classifier_name_epl_2_0() -> None:
```
Comment (Secrus) [inline tests/spdx/test_license.py:15]: I don't think this test case makes much sense. I get that it checks your exact issue case, but I don't see why we should check this exact license among others that we support. I am ok with no test case for that and treat it as a simple data update.

Comment (netomi) [reply tests/spdx/test_license.py:15]: I am fine with leaving out a testcase for such a trivial test. I just wanted to check myself that my change resulted in the correct license name that is later used for generating the classifier.

**Comment by netomi** [comment]
Added a test case to ensure that all known licenses return a valid trove classifier.
While adding the test I figured that the Aladdin license was also failing, so fixed it right away.

**Review by Secrus — APPROVED** [review]

**Pull request merged**